src/widget/wtracktableview.cpp

```diff
@@ -503,6 +503,10 @@ void WTrackTableView::createActions() {
     connect(m_pClearLoopAction, SIGNAL(triggered()),
             this, SLOT(slotClearLoop()));
 
+    m_pClearKeysAction = new QAction(tr("Key(s)"), this);
```

Comment:
I think the text should be singular ("Key"). We hope to support multiple BPMs in the future too, but resetting "BPMs" feels awkward IMO.